gcs/INSTALL.md

```diff
@@ -37,14 +37,41 @@ Once you have the JSON key file, you must add the following property to
 `core-site.xml` on your server.
 
 ```xml
+<property>
+  <name>fs.gs.project.id</name>
+  <value></value>
+  <description>
+      Optional. Google Cloud Project ID with access to GCS buckets.
+      Required only for list buckets and create bucket operations.
+  </description>
+</property>
+<property>
+  <name>google.cloud.auth.service.account.enable</name>
+  <value>/path/to/keyfile</value>
+  <description>
+      Whether to use a service account for GCS authorization.
+      If set to `false` then GCE VM metadata service will be used
+      for authorization.
+  </description>
```

Comment:
Alternatively you can set GOOGLE_APPLICATION_CREDENTIALS=/path/to

Add that to the description.